pkgs/tools/graphics/blockhash/default.nix

```diff
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
   strictDeps = true;
 
   meta = with lib; {
-    homepage = "http://blockhash.io/";
+    homepage = "https://blockhash.io/";
```

Comment:
I saw "BlockHash.io is for sale" :smile:

Reply:
Rip, should I link to archive.org?

Reply:
Maybe https://github.com/commonsmachinery/blockhash ?

Reply:
Ok!